build.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
```

Comment:
Lets have all three then? Can you add the one for PS? I don't see these files growing anytime soon. Build logic should really go into build.fsx.